prevent wpaclean from overwriting the outputfile
with next capture each time a new capture is found
in a source file

Index: src/wpaclean.c
--- src/wpaclean.c.orig
+++ src/wpaclean.c
@@ -138,7 +138,10 @@ static void save_network(const struct network * n)
 {
 	int i;
 
-	_outfd = open_pcap(_outfilename);
+	if (_outfd == 0)
+	{
+		_outfd = open_pcap(_outfilename);
+	}
 	write_pcap(_outfd, n->n_beacon, n->n_beaconlen);
 
 	for (i = 0; i < 4; i++)
